.buildkite/claim-settlements.yml

```diff
@@ -67,12 +67,15 @@ steps:
     - '. "$HOME/.cargo/env"'
     - 'buildkite-agent artifact download --include-retried-jobs target/release/claim-settlement .'
     - 'chmod +x target/release/claim-settlement'
+    - 'echo "UNKNOWN ERROR" > ./claiming-report.txt'
```

Comment:
leftover?

Reply:
Yes, or maybe rather an error. Idea behind was to have reported an `unknown error` when something wrong happens on call claim settlement before it goes running and to ensure there is the artifact existing for further use. But as the `tee` is creating new file either way this should be ensured. Leaving this as it is as it is changed in follow-up PR.
Thanks for the feedback here.